keyboards/keebio/bfo9000/keymaps/via/keymap.c

```diff
@@ -0,0 +1,39 @@
+#include QMK_KEYBOARD_H
```

Comment:
Needs a gpl compliant header, for example

```suggestion
// Copyright 2022 John-Schreiber
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```